### PR TITLE
Uniques: Small fix in docs

### PR DESCRIPTION
**Pull request by Szegoo**
Was reading through the code and noticed this minor inaccuracy in the docs.

### CODE REVIEW

File: frame/uniques/src/lib.rs [hunk 1]
```diff
@@ -908,7 +908,7 @@ pub mod pallet {
 
 		/// Approve an item to be transferred by a delegated third-party account.
 		///
-		/// Origin must be Signed and must be the owner of the `item`.
+		/// Origin must be Signed and must be the owner of the `item` or the admin.
```
Comment (bkchr) [inline frame/uniques/src/lib.rs:911]:
```suggestion
		/// The origin must conform to `ForceOrigin` or must be `Signed` and the sender must be the
		/// owner of the `item`.
```

**Review by bkchr — APPROVED** [review]

File: frame/uniques/src/lib.rs (hunk 1 above)
Comment (Szegoo) [reply frame/uniques/src/lib.rs]: Actually, this is not what I meant, in the function, there is a check whether the origin is the owner of the item or if the origin is the admin of the collection.

Comment (bkchr) [reply frame/uniques/src/lib.rs]: Changed the wording. We should also include `ForceOrigin`. As you see this is also done in the other docs here.

**Review by kianenigma — APPROVED** [review]

**Comment by kianenigma** [comment]
bot merge

**Pull request merged**